mobile/ios/scripts/build_arch.sh

```diff
@@ -13,10 +13,18 @@ if [ "$PLATFORM" = "iPhoneOS" ]; then
     EXTRA_CONFIG="--host=arm-apple-darwin14 --target=arm-apple-darwin14"
 fi
 
+ROOTDIR=$(cd $(dirname $0) && pwd -P)
+SOURCEDIR="$ROOTDIR/../../../"
+BUILDDIR="$ROOTDIR/../"
+
+#We assume that the user has the iphoneos sdk installed
```

Comment:
This should read `# XXX We assume...` since it's an unfriendly assumption (if the SDK is not installed, this fails in an obscure way and the developer is not correctly informed)